src/transformers/modeling_utils.py

```diff
@@ -105,7 +105,7 @@
 XLA_DOWNCAST_BF16 = os.environ.get("XLA_DOWNCAST_BF16", "0").upper()
 
 if is_accelerate_available():
-    from accelerate import dispatch_model, infer_auto_device_map, init_empty_weights
+    from accelerate import Accelerator, dispatch_model, infer_auto_device_map, init_empty_weights
```

Comment:
Let's just use `extract_model_from_parallel` instead of going through the `Accelerator`, since that's all it's calling

Reply:
Yeah I thought about that too. Maybe, we can put that in the `unwrap_model` so that it is easier to understand. However, we need to add a test to make sure that we have the same behavior.